src/Microsoft.ApplicationInsights/Extensibility/W3C/W3CConstants.cs

```diff
@@ -1,60 +1,13 @@
 ﻿namespace Microsoft.ApplicationInsights.Extensibility.W3C
 {
+    using System;
+    using System.Diagnostics;
+
     /// <summary>
     /// W3C constants.
     /// </summary>
     internal static class W3CConstants
     {
-        /// <summary>
-        /// Trace-Id tag name.
-        /// </summary>
-        internal const string TraceIdTag = "w3c_traceId";
-
-        /// <summary>
-        /// Span-Id tag name.
-        /// </summary>
-        internal const string SpanIdTag = "w3c_spanId";
-
-        /// <summary>
-        /// Parent span-Id tag name.
-        /// </summary>
-        internal const string ParentSpanIdTag = "w3c_parentSpanId";
-
-        /// <summary>
-        /// Version tag name.
-        /// </summary>
-        internal const string VersionTag = "w3c_version";
-
-        /// <summary>
-        /// Sampled tag name.
-        /// </summary>
-        internal const string SampledTag = "w3c_sampled";
-
-        /// <summary>
-        /// Tracestate tag name.
-        /// </summary>
-        internal const string TracestateTag = "w3c_tracestate";
-
-        /// <summary>
-        /// Default version value.
-        /// </summary>
-        internal const string DefaultVersion = "00";
-
-        /// <summary>
-        /// Default sampled flag value: may be recorded, not requested.
-        /// </summary>
-        internal const string TraceFlagRecordedAndNotRequested = "02";
-
-        /// <summary>
-        /// Recorded and requested sampled flag value.
-        /// </summary>
-        internal const string TraceFlagRecordedAndRequested = "03";
-
-        /// <summary>
-        /// Requested trace flag.
-        /// </summary>
-        internal const byte RequestedTraceFlag = 1;
-
```

Comment:
is `LegacyRequestIdProperty` still used anywhere? I guess no and it can be removed